integration.md

```diff
@@ -81,30 +81,32 @@ We've previously stated that the forms presented by Klarna are eventually posted
 ```ruby
 # Handle POST requests to '/purchase'
 post '/purchase' do
+  @user = User.create(session, params[:userToken], 'user@email.com') unless @user
   purchase = Klarna::Purchase.new(
-    user_token:    params[:userToken],
-    reference:     "IA-#{params[:article_id]}",
-    name:          'Interesting Article',
+    user_token:    @user.token,
```

Comment:
@grozen would you prefer I'll keep the params[:userToken] + "IA-#{params[:article_id]}" + 'Interesting Article' instead of using the User constructor?
cc: @yuval-netanel